### PR TITLE
Reflect production prefix names

### PR DESCRIPTION
**Pull request by MalloZup**
We need a 1:1 production, so people learn right prefixes names

### CODE REVIEW

**Comment by moio** [comment]
Too bad this is inconsistent with all other defaults in `sumaform` - why were different names chosen? Eg. I expect most developers to be familiar with `minsles12sp2`.

**Comment by MalloZup** [comment]
we took a decision according to this:
https://github.com/SUSE/spacewalk/wiki/Cucumber-Sumaform-Infrastracture#new-naming-scheme

**Comment by moio** [comment]
Then why `srv` and not `suma-head-srv`?

**Comment by MalloZup** [comment]
"suma-head" is the prefix in base module.
it can be then
"dma-srv"

**Comment by moio** [comment]
From my perspective the prefix serves a different purpose, as you explicitly configure `srv` to have `version = "head"`

**Comment by MalloZup** [comment]
@moio  i was not the only one that took the decision. Maybe we misbehaved the Prefix, but the goal was to have
 "suma" (production prefix)
  -head ( version)
  -srv (role)
etc. 
At the time it was more like the highlevel and we overlooked the difference vs sumaform.

Imho now t will take time to change the names on prod

**Comment by moio** [comment]
> @moio i was not the only one that took the decision. [...]

This is irrelevant to my last comment. I am saying that `srv` is wrong, should be `suma-head-srv` or at least `head-srv` as it is specified in the example file that the version is `head`.

**Comment by MalloZup** [comment]
ok i got what you mean, thx

**Comment by MalloZup** [comment]
@moio ok now i added the prefix "head".

thx!

**Comment by moio** [comment]
Are you OK with the [latest change](1deca67a3ab767b1b1ed9a1131d2f4cb762406c3) I'm proposing here?

**Comment by MalloZup** [comment]
sure

@moio thx!

File: main.tf.libvirt-testsuite.example [hunk 1]
```diff
@@ -10,68 +10,68 @@ module "base" {
...
   // you need always sles12sp1 because controller is based on that.
   images = ["centos7", "sles12sp1", "sles12sp2"]
 }
 
-module "srv" {
+module "head-srv" {
   source = "./modules/libvirt/suse_manager"
   base_configuration = "${module.base.configuration}"
-  name = "srv"
+  name = "head-srv"
```
Comment (MalloZup) [inline main.tf.libvirt-testsuite.example]: ok

**Pull request merged**